src/Model/Svg/SvgFile.php

```diff
@@ -470,6 +471,17 @@ protected function analyse(): void
         $this->savedLanguages = array_unique($this->savedLanguages);
     }
 
+    /**
+     * Workaround for O(N^2) performance of DOMDocument::getElementsByTagName() T336917
+     *
+     * @param string $name
+     * @return DOMNodeList
+     */
+    private function getElementsByTagName(string $name): DOMNodeList
+    {
+        return $this->xpath->query("//*[name()=\"$name\"]");
```

Comment:
This fails the tests, because `name()` matches the whole name including namespace.

```
There was 1 failure:

1) App\Tests\Model\Svg\SvgFileTest::testSvgNamespace with data set "text_namespace" ('<switch><svg:text id="trsvg18...witch>', '<switch><svg:text id="trsvg18...witch>')
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
 '<?xml version="1.0" encoding="UTF-8"?>
-<svg xmlns:svg="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg"><switch><svg:text id="trsvg18-zh-hans" systemLanguage="zh-hans"><tspan id="trsvg2-zh-hans">A</tspan></svg:text><svg:text id="trsvg18"><tspan id="trsvg2">B</tspan></svg:text></switch></svg>
+<svg xmlns:svg="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg"><switch><svg:text id="trsvg18-zh-hans" systemLanguage="zh-Hans"><tspan id="trsvg2-zh-hans">A</tspan></svg:text><svg:text id="trsvg18"><tspan id="trsvg2">B</tspan></svg:text></switch></svg>
 '
```

It looks like switching to `local-name()` works as expected.